Makefile

```diff
@@ -27,6 +29,13 @@ test: deps $(SOURCES)
 	PATH=$(CURRENT_DIR)/bin:$(PATH) go test $(PACKAGES) $(TESTARGS)
 	go vet $(PACKAGES)
 
+gometalint-exists:
+	@which gometalinter > /dev/null || \
```

Comment:
Agree. You can add it to [`deps`](https://github.com/allegro/marathon-consul/blob/3607a6be4c1bba0843d42dcf62886e4a46f13183/Makefile#L14) target